docs/reference/commandline/network_create.md

```diff
@@ -175,7 +175,7 @@ equivalent docker daemon flags used for docker0 bridge:
 | `com.docker.network.bridge.enable_icc`           | `--icc`     | Enable or Disable Inter Container Connectivity        |
 | `com.docker.network.bridge.host_binding_ipv4`    | `--ip`      | Default IP when binding container ports               |
 | `com.docker.network.driver.mtu`                  | `--mtu`     | Set the containers network MTU                        |
-| `com.docker.network.container_interface_prefix`  | -           | Set a custom prefix for container interfaces          |
+| `com.docker.network.container_iface_prefix`      | -           | Set a custom prefix for container interfaces          |
```

Comment:
Hmmm.. good catch; looks indeed the correct name; https://github.com/moby/moby/blob/a8a769f04f7ae44117772943ef0f4607b54e05ff/vendor/github.com/docker/libnetwork/netlabel/labels.go#L54-L55

```go
// ContainerIfacePrefix can be used to override the interface prefix used inside the container
ContainerIfacePrefix = Prefix + ".container_iface_prefix"
```

Looks like https://github.com/moby/moby/pull/31811 updated the changelog incorrectly, probably because the title of the libnetwork pull request had the wrong name in it (I just fixed that); https://github.com/moby/libnetwork/pull/1667